THIRD_PARTY.md

```diff
@@ -0,0 +1,9 @@
+# Python Dependencies
+
+## Build Dependencies List
+
+See [Pre-Build Requirements](build_requirements.txt) and [Build Requirements](setup.py)
```

Comment:
To save folks the effort of having to navigate `setup.py`, could we be more specific and jump them to the dependency list, like: [Build Requirements](https://github.com/elyra-ai/elyra/blob/a8fb8cbde883e16cffb74329ed54bf8275668f5f/setup.py#L66)?  Hmm, I suppose a "permalink" is too permanent for this, so perhaps just a file reference is the best we can do.

(We should also move to `pyproject.toml` at some point - perhaps a roadmap thing for next year?)

Reply:
Yeah roadmap the pyproject.toml improvement. The file reference I feel should be adequate. Developers should know how to read and find the dependencies in a setup.py.